test/api_container_create_test.go

```diff
@@ -279,3 +280,86 @@ func (suite *APIContainerCreateSuite) TestCreateWithMacAddress(c *check.C) {
 
 	DelContainerForceMultyTime(c, cname)
 }
+
+func (suite *APIContainerCreateSuite) TestBasicWithSpecificID(c *check.C) {
+	{
```

Comment:
missing test case. specificID is conflict with an existing container's ID

Reply:
add test for it!